DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/BulkUpload/StartUpload.cshtml

```diff
@@ -21,27 +20,13 @@
     <h1 class="nhsuk-heading-xl">Bulk upload/update delegates</h1>
 
     <form class="nhsuk-u-margin-bottom-3" method="post" asp-action="StartUpload" enctype="multipart/form-data">
-      <div class="nhsuk-checkboxes nhsuk-checkboxes--conditional nhsuk-u-margin-bottom-4">
-        <div class="nhsuk-checkboxes__item">
-          <input class="nhsuk-checkboxes__input"
-                 asp-for="ShouldSendEmail"
-                 type="checkbox"
-                 value="true"
-                 aria-controls="@dateInputId"
-                 aria-expanded="@Model.ShouldSendEmail">
-          <label class="nhsuk-label nhsuk-checkboxes__label" asp-for="ShouldSendEmail">
-            Send welcome email to registered delegates
-          </label>
-        </div>
-
-        <vc:date-input id="@dateInputId"
-                       label="Deliver email on or after"
-                       day-id="Day"
-                       month-id="Month"
-                       year-id="Year"
-                       css-class="@emailDateCss"
-                       hint-text-lines="@hintTextLines" />
-      </div>
+      <vc:date-input id="@dateInputId"
+                     label="Send welcome email to registered delegates, to be delivered on or after"
+                     day-id="Day"
+                     month-id="Month"
+                     year-id="Year"
+                     css-class="@(errorHasOccurred ? " nhsuk-u-padding-left-5 nhsuk-u-margin-bottom-3" : "")"
+                     hint-text-lines="@hintTextLines" />

```

Comment:
Spotted another couple issues here when checking that the new margin looked okay - 
1. When I enter an invalid date and do not upload a file, and then submit, I get an error on the file input but not the date input.
2. When I enter an invalid date and _do_ upload a file, and then submit, I get the error on the date but the file disappears.

Reply:
1. happens because the date validation code is not called if there's a missing file error. I've talked to @SteveJacksonSoft about a potential fix, but we think it's not really worth it because you'll get the error for the date eventually anyway, and people using this interface will usually be adding a file.

2. happens because we don't (and possibly can't) store the file in TempData. I don't think we should try to resolve it.

Reply:
@simonbrent Maybe leave a comment on the ticket so HEE testers know that's expected behaviour

Reply:
Ok